Plan.md

```
@@ -220,3 +220,31 @@ docs.rs のドキュメントが簡略化されており、実際の API 詳細
+    - 42MB の Image 生成
+  - Data Abort ハンドラーの IPA 取得バグ修正
+    - 問題: FAR_EL1 から 0 が返され、フォールバックで X1 の値（書き込み文字）をアドレスとして使用していた
+    - 原因: Stage 2 フォールトでは FAR_EL1 ではなく exit_info.exception.physical_address を使用すべき
+    - 修正: handle_data_abort() に fault_ipa 引数を追加し、physical_address を使用
+  - 未知のシステムレジスタアクセスをエミュレート
+    - 読み取り時は 0 を返し、書き込み時は無視して続行
+  - linux_boot_test.rs 作成
+    - UartCollector で UART 出力をキャプチャ
+    - 3904 バイトの正常な起動ログを確認
+  - 確認できた起動ログ
+    ```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)